README.md

```diff
@@ -158,9 +158,9 @@ See `example.php` for further examples.
 
 ## User Impersonation
 
-As of Redmine V2.2 you can impersonate user through the REST API :
+As of Redmine V2.2, you can impersonate user through the REST API :
 
-```php
+PHP
```

Comment:
:-1: 
this line marks a code area with PHP syntax highlighting. This change makes no sense.